bootstrap/src/main/scala/org/scalafmt/bootstrap/Bootstrap.scala

```diff
@@ -41,11 +41,14 @@ object ScalafmtBootstrap {
   private val cliCache =
     mutable.Map.empty[String, Either[Throwable, ScalafmtBootstrap]]
 
-  def fromVersion(version: String): Either[Throwable, ScalafmtBootstrap] =
-    cliCache.getOrElseUpdate(version, fromVersionUncached(version))
+  def fromVersion(version: String): Either[Throwable, ScalafmtBootstrap] = {
+    synchronized {
```

Comment:
I just noticed this line, do you think it's necessary? We already synchronize on `fromVersionUncached`.